tests/test_decompress.py

```
@@ -1262,6 +1462,354 @@ def test_agent_name_parsing(self):
+        has_price, cost, is_estimated = self._cost_row(1)
+
+        self.assertEqual(has_price, 1)
+        self.assertAlmostEqual(cost, 2.0 + 8.0)
+        self.assertEqual(is_estimated, 1)
+
+    def test_specific_price_beats_provider_catch_all(self):
+        self._insert_price("acme-sub", "", input_price=0, output_price=0, is_estimated=True)
+        self._insert_price("acme-sub", "premium-model", input_price=5.0, output_price=5.0)
+        self._insert_usage(1, "acme-sub", "premium-model", input_tokens=1_000_000)
+
+        has_price, cost, is_estimated = self._cost_row(1)
```

Ruff (0.16.2)

[warning] Unpacked variable `has_price` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)